src/main.rs

```diff
@@ -486,7 +486,13 @@ impl Halloy {
                         dashboard
                             .broadcast(&server, &self.config, sent_time, Broadcast::Reconnected)
                             .map(Message::Dashboard)
-                    }
+                    };
+
+                    let focus_focused_pane_buffer = dashboard
+                        .focus_focused_pane_buffer()
+                        .map(Message::Dashboard);
+
+                    Task::batch(vec![broadcast, focus_focused_pane_buffer])
```

Comment:
Why refocus here? Is focus taken away somehow or are we using this as the initial thing to focus the pane? Can we "refocus" instead when first entering dashboard screen?